scripts/scil_compute_local_tracking_dev.py

```diff
@@ -186,7 +176,7 @@ def main():
     logging.debug("Instantiating tracker.")
     tracker = Tracker(propagator, mask, seed_generator, nbr_seeds, min_nbr_pts,
                       max_nbr_pts, max_invalid_dirs, args.compress,
-                      args.nbr_processes, args.save_seeds, mmap_mode,
+                      args.nbr_processes, args.save_seeds, mmap_mode='r+',
```

Comment:
In the Tracker, could we improve the description of mmap_mode. Ex: add suggestions, explanations.

Reply:
@GuillaumeTh @CHrlS98 ? any suggestions ?

Reply:
Right now it reads
```
mmap_mode: str
            Memory-mapping mode. One of {None, ‘r+’, ‘r’, ‘w+’, ‘c’}
```
First, @GuillaumeTh and I ran some tests and options 'r' and 'w+' result in a crash, so they shouldn't be in the choices. The descriptions for the remaining mmap mode can be found here https://numpy.org/doc/stable/reference/generated/numpy.memmap.html#numpy.memmap. I'd just copy the corresponding descriptions and say it tells the tracker how to access the data across multiple threads. Maybe we should add a safety check in the `__init__` to validate that memmap is one of {None, 'r+', 'c'}.

Reply:
Can I put 'r+' as default mmap_mode for the Tracker ? @CHrlS98 @GuillaumeTh @EmmaRenauld  ?